arlo_server/__init__.py

```diff
@@ -16,7 +17,7 @@
     # Restrict which hosts we trust when not in dev/test. This works by causing
     # anything accessing the request URL (i.e. `request.url` or similar) to
     # throw an exception if it doesn't match one of the values in this list.
-    Request.trusted_hosts = [HTTP_ORIGIN]
+    Request.trusted_hosts = [str(urlparse(HTTP_ORIGIN).hostname)]
```

Comment:
Have to wrap with `str()` because the type of `.hostname` is `Optional[str]`. Dunno if anyone knows a better way to deal with this.